pubspec.yaml

```diff
@@ -1,53 +1,56 @@
 name: at_dude
 description: atDude App
-version: 1.0.3+6
+version: 1.0.0+3
```

Comment:
Version number regressed?

Reply:
I'll actually reset it to 1.0.0 when it's ready to be released. I just change the numbers to not have a conflict with the different builds on test flight.